packages/@lwc/engine-core/src/testFeatureFlag.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import features from '@lwc/features';
+
+if (process.env.NODE_ENV !== 'production') {
+    if (features.ENABLE_TEST_EXCEPTION) {
```

Comment:
How is this test suppose to work?

Reply:
I also had a question about this, when would `features.ENABLE_TEST_EXCEPTION` be set to true?

Reply:
It's never set to true. The point is just that, if the compilation pipeline is broken, then this will throw an error. It was previously throwing an error because `@lwc/engine-dom`'s Rollup scripts were not configured properly.

I can add a comment.

Reply:
> The point is just that, if the compilation pipeline is broken, then this will throw an error.

Was this throwing the error at runtime?

I tried to insert these statements and run `yarn build` locally and it didn't throw. Test Branch: https://github.com/salesforce/lwc/tree/ravi/test-feature-flag

Reply:
In your PR, the feature flags don't actually work. 🙂  If you try doing `setFeatureFlagForTest` then nothing will happen, because the LWC features Babel plugin is not being applied to the code.

Whereas if you _do_ apply the Babel plugin, it breaks because it runs twice on both `@lwc/engine-dom` and `@lwc/engine-core`... that's what this PR is fixing.

Reply:
Makes sense. Thanks for clarifying.